_gsocprojects/2019/project_AstroLab.md

```diff
@@ -3,9 +3,9 @@ project: AstroLab
 layout: default
 logo: Icone_AstroLab_sans_fond.png
 description: |
-  [AstroLab Software](https://astrolabsoftware.github.io/) is a project from [LAL](www.lal.in2p3.fr) aiming at providing advanced software tools to overcome modern science challenges faced by research groups, and allow research communities to more fully exploit the big data ecosystem tools.
+  [AstroLab Software](https://astrolabsoftware.github.io/) is a project from [LAL](https://www.lal.in2p3.fr/en/) aiming at providing advanced software tools to overcome modern science challenges faced by research groups, and allow research communities to more fully exploit the big data ecosystem tools.
```

Comment:
The problem with `https` was indeed fixed on a previous PR, so it leads to a conflict now. Can you please solve it?